src/models/whatsnewmodel.cpp

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "whatsnewmodel.h"
+#include "leakdetector.h"
+#include "logger.h"
+#include "mozillavpn.h"
+#include "settingsholder.h"
+#include "featurelist.h"
+#include "feature.h"
+
+#include <QList>
+
+namespace {
+Logger logger(LOG_MODEL, "WhatsNewModel");
+}
+
+WhatsNewModel::WhatsNewModel() {
+  MVPN_COUNT_CTOR(WhatsNewModel);
+
+  WhatsNewModel::setNewFeatures();
```

Comment:
Let's do it lazily:
1. add a `bool m_initialized = false;`
2. add a private method: `void maybeInitialize()`;
3. call the `maybeInitialize()` method in `featureCount()`, `roleNames()`, `rowCount()` and `data()`.
4. in `maybeInitialize()` does something like this:

```
if (m_initialized) { return; }

m_initialized = true;
// the code you have in setNewFeatures() here
```